src/components/site/SiteHeader.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 const SiteHeader = props => {
```

Comment:
not blocking but in this case, this can be written.

```jsx
const SiteHeader = props => <header className="header">{props.children}</header>;
```

The case for this format is when you do not need to restructure the props, like in this case.

Reply:
changed format